doc/contrail_deployment/contrail_deployment_manual.md

```diff
@@ -0,0 +1,38 @@
+# How to deploy Contrail with Windows compute nodes
+
+## Prerequisites
+
+* Machine with CentOS 7.4 installed
+* Machine(s) with Windows Server 2016 and updates installed
+* Machine for running Ansible playbooks (Linux or Windows with WSL)
+
+## Steps
+
+* On the Ansible machine:
+  * `git clone git@github.com:Juniper/contrail-ansible-deployer.git`
+  * `cd contrail-ansible-deployer`

```

Comment:
Please add pip install -r python-requirements.txt

Reply:
We will think about installing it during `configure_instances`